approaches/transformers.md

```diff
@@ -1,7 +1,41 @@
 ---
 grand_parent: Approaches
 parent: Neural machine translation
-layout: coming_soon
 title: Transformers
 description: A deep learning model architecture for neural machine translation
 ---
+
+**Transformers** are a type of neural network architecture that relies only on the [attention](/concepts/attention.md) mechanism.
+It allows [neural machine translation](/approaches/neural-machine-translation.md) models to better handle the dependencies between the input and output sequences.
+
+The Transformer was first introduced in a 2017 paper by Google researchers titled _"Attention Is All You Need."_
+
+## Architecture
+
+The Transformer consists of an encoder and a decoder.
+Each consists of several layers, which include sublayers of *self-attention* and *feed-forward neural networks*.
```

Comment:
To minimize potential quality problems when we machine translate Machine Translate, let's avoid reference words like "each". In this case, it's best to repeat the subject. :)